scripts/add-cluster.sh

```diff
@@ -224,10 +230,16 @@ echo ${CLUSTER_JOIN_TO_OPERATOR_NS}
 login_to_cluster ${JOINING_CLUSTER_TYPE}
 
 if [[ ${JOINING_CLUSTER_TYPE_NAME} != "e2e" ]]; then
-    SA_NAME="toolchaincluster-${JOINING_CLUSTER_TYPE}-operator"
+    SA_NAME="toolchaincluster-${JOINING_CLUSTER_TYPE}"
+    if [[ ! -z ${MULTI_MEMBER} ]]; then
+      SA_NAME="${OPERATOR_NS}"
+    fi
```

Comment:
couldn't you do the same as for `ToolchainCluster` CRs?
```suggestion
    SA_NAME="toolchaincluster-${JOINING_CLUSTER_TYPE}${MULTI_MEMBER}"
```

Reply:
That's true, if -mm is not used then ${MULTI_MEMBER} should be empty and will result in the same naming pattern before this PR so your suggestion should work.